extensions/markdown-language-features/src/features/preview.ts

```diff
@@ -227,6 +227,10 @@ export class DynamicMarkdownPreview extends Disposable {
 			}
 		}));
 
+		this._register(vscode.workspace.onDidChangeConfiguration(_ => {
```

Comment:
We should not require any changes to extension code. Remember, this effects more than just the MD preview. We can't require every webview extension to have this logic